static/style.css

```diff
@@ -48,6 +48,7 @@ code {
 
 .plot-img {
   width: 80%;
+  image-rendering: pixelated;
```

Comment:
I'm not sure it makes sense to turn this on globally. This makes images crisp when they get scaled up, which was useful for figuring out antialiasing and such (since the images are always upscaled to 80% width).